lib/hipe/rtl/Makefile

```diff
@@ -135,7 +135,7 @@ HIPE_MKLITERALS=$(ERL_TOP)/bin/$(TARGET)/hipe_mkliterals$(TYPE_STR)$(FLAVOR_STR)
 
 
 hipe_literals.hrl: $(HIPE_MKLITERALS)
-	$(gen_verbose)$(HIPE_MKLITERALS) $(MKLIT_FLAGS) -e > hipe_literals.hrl
+	$(gen_verbose)$(RUN) $(HIPE_MKLITERALS) $(MKLIT_FLAGS) -e > hipe_literals.hrl
```

Comment:
Yes! If RUN is not set, it's as if it isn't there; it will just run `hipe_mkliterals` directly. As you can see, the travis build, which is without RUN, passed fine.